File: IMS/IMS.Presentation/DTOs/GetDTO/BoardDTO.cs
namespace IMS.Presentation.DTOs.GetDTO;

public class BoardDto
{
    public Guid Id { get; init; }
    public required string Title { get; init; } 
    public required string Description { get; init; } 
    public Guid CreatedById { get; init; }  
    public Guid CreatedToId { get; init; }
}

File: Frontend/ims-frontend/src/entities/ims/dto/board_dto.ts
export interface BoardDto {
    id: string;
    title?: string | null;
    description?: string | null;
    createdById: string;
    createdToId: string;
}

export interface CreateBoardDto {
    createdById: string
    createdToId: string;
    title?: string | null;
    description?: string | null;
}

export interface UpdateBoardDto {
    title?: string | null;
    description?: string | null;
}
